google/resource_bigquery_table.go

```diff
@@ -394,3 +428,24 @@ func flattenTimePartitioning(tp *bigquery.TimePartitioning) []map[string]interfa
 
 	return []map[string]interface{}{result}
 }
+
+func expandView(configured interface{}) *bigquery.ViewDefinition {
+	raw := configured.([]interface{})[0].(map[string]interface{})
+	vd := &bigquery.ViewDefinition{Query: raw["query"].(string)}
+
+	if v, ok := raw["use_legacy_sql"]; ok {
+		vd.UseLegacySql = v.(bool)
```

Comment:
You'll also need to set `vd.ForceSendFields = append(vd.ForceSendFields, "UseLegacySql"` so if it gets updated to false, it still gets sent along the wire (since false values end up defaulting to empty)